### DISCUSSION TITLE
TeX/LaTeX processing not supported on GitHub mobile

### DISCUSSION THREAD

**Discussion by imraklr** [community/community#136519] [Mobile]
I have some colored text in my repositories README coded in LaTeX. This colored text is visible on desktop web browsers but not on my mobile. Instead of colors it shows code.

**Comment by eliperkins** [comment]
Hey @imraklr, thanks for this feedback! I've added your +1 to our internal tracking issue for this which was [created for this original discussion post](https://github.com/orgs/community/discussions/18128). I'll work with the engineering team to figure out a path forward to get this addressed.

If you could add your upvote to the [original discussion post for LaTeX support on Mobile](https://github.com/orgs/community/discussions/18128), that'll help give us a better signal in one singular issue!
